Plots/ColormapComparisons/CB_Rain.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jun  4 14:12:34 2020
+
+@author: misi1684
+"""
+
+
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+from geocat.viz import util as gvutil
+import cartopy.feature as cfeature
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/pre.8912.mon.nc"))
+
+# Extract a slice of the data
+t = ds.pre[0,:]
+
+#gvutil.xr_add_cyclic_longitudes(t,'lon')
```

Comment:
You commented this out. Does it still need to be here, or can it be removed? If it needs to remain, add a comment explaining what happens with and without this line.